rpcq/core_messages.py

```diff
@@ -694,7 +711,7 @@ class QFDChannel(Message):
     flux_current: Optional[float] = None
     """Flux current [Amps]."""
 
-    relay_closed: Optional[bool] = None
+    relay_closed: Optional[bool] = False
```

Comment:
Unfortunately auto-generation is broken for this. Can you manually set it back to `None`?

Reply:
My apologies.  When the auto-gen ran I just assumed it correct.  Changing it now.